tajo-catalog/tajo-catalog-common/src/main/java/org/apache/tajo/catalog/exception/NoSuchPartitionException.java

```diff
@@ -32,6 +32,10 @@ public NoSuchPartitionException(String message) {
     super(message);
   }
 
+  public NoSuchPartitionException(String tableName, String partitionName) {
```

Comment:
The imports in this source code are unused. Please remove them. I know that this is not related to your changes.